### PR TITLE
fix: clear some console warnings

### PR DESCRIPTION
**Pull request by oranges-eating**
Signed-off-by: killer <1533063601@qq.com>

### CODE REVIEW

**Comment by kocoler** [comment]
LGTM

File: web/src/main/BalanceBox.js [hunk 1]
```diff
@@ -99,7 +99,7 @@ class BalanceBox extends React.Component {
     switch (record?.consumptionType) {
       case 1:
         return (
-          <tr>
+          <tr key={record.createdTime}>
```
Comment (hsluoyz) [inline web/src/main/BalanceBox.js:102]: Don't use `createdTime`, `createdTime` can be the same.

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: web/src/main/BalanceBox.js (hunk 1 above)
Comment (oranges-eating) [reply web/src/main/BalanceBox.js:102]: > Don't use `createdTime`, `createdTime` can be the same.

捕获
Record has these attributes, I think createdTime is the least likely to be repeated

**Comment by oranges-eating** [comment]
@hsluoyz What's wrong with this pr?

**Pull request closed without merging**